docs/man-openmpi/man3/ERRORS.rst

```diff
@@ -0,0 +1,33 @@
+Almost all MPI routines return an error value; C routines as the return result
+of the function and Fortran routines in the last argument.
+
+Before the error value is returned, the current MPI error handler is called.
+If no other approriate error handler has been set, then the default error
+handler, MPI_ERRORS_ARE_FATAL, is called.
+
+The default error handler aborts the MPI application, except for I/O function
+errors.
+
+MPI applications can set error handlers on communicators, files, windows, and
+sessions.
+
+In the sessions model, the error handler can be set during MPI_Session_init.
+
+Open MPI includes three predefined error handlers that can be used:
+
+MPI_ERRORS_ARE_FATAL: Causes the program to abort all connected MPI processes.
+MPI_ERRORS_ABORT: An error handler that can be invoked on a communicator,
+                  window, file, or session. When called on a communicator, it
+                  acts as if MPI_ABORT was called on that communicator. If
+                  called on a window or file, acts as if MPI_ABORT was called
+                  on a communicator containing the associated processes. If
+                  called on a session, aborts only the local processes.
+MPI_ERRORS_RETURN: Returns an error code to the application.
+
+MPI applications can also implement their own error handlers.
+
+Custom MPI error handlers can be created by calling one of the
+MPI_XXX_CREATE_ERRHANDLER functions. Predefined and custom error handlers can
+be set by calling one of the MPI_XXX_SET_ERRHANDLER functions.
+
+See the Error Handling section of the MPI-4 standard for more information.
```

Comment:
```suggestion
See the Error Handling section of the MPI-|mpi_standard_version| standard for more information.
```

This a Sphinx macro we (OMPI) defined that will automatically substitute in the MPI standard major version that this release of Open MPI supports.  You can use `|mpi_standard_version|.|mpi_standard_subversion|` if you want the full "major.minor" version of the MPI standard that this release of Open MPI supports.